src/applications/check-in/pre-check-in/tests/e2e/errors/post-session/error.in.body.cypress.spec.js

```diff
@@ -31,7 +31,7 @@ describe('Pre-Check In Experience ', () => {
 
       ValidateVeteran.attemptToGoToNextPage();
 
-      Error.validatePageLoaded();
+      Error.validatePageLoaded(true, true);
```

Comment:
This seems a little bit weird - do the cypress fixtures default to sending phone appointments now?

Reply:
This is a stop gap, John has an in-progress PR to address this.